### DISCUSSION TITLE
在docker进行服务化部署，发送http请求，出错

### DISCUSSION THREAD

**Discussion by penghui-luna** [PaddlePaddle/PaddleOCR#12666] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：CUDA Version: 11.6   
- 版本号/Version：Paddle：paddle-gpu=2.3.2  PaddleOCR：2.5 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
- {
    ![image](https://github.com/PaddlePaddle/PaddleOCR/assets/40514537/9163c485-d9f4-4b1b-a5d4-0305bc344195)
}

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by vicky2399** [comment]
hi i like to assign the issue
